allauth/account/adapter.py

```diff
@@ -243,7 +243,7 @@ def is_open_for_signup(self, request):
         Next to simply returning True/False you can also intervene the
         regular flow by raising an ImmediateHttpResponse
         """
-        return True
+        return app_settings.ALLOW_SIGNUP
```

Comment:
Naming wise this is inconsistent with the adapter method, would have accepted IS_OPEN_FOR_SIGNUP?